bundles/org.openhab.binding.sunsynk/README.md

```diff
@@ -0,0 +1,199 @@
+See [Data Logger set up](https://www.sunsynk.org/_files/ugd/39fbfb_a325b6884e684c4ba1a3ad80afd5da20.pdf) or [Sun Synk Web](https://www.sunsynk.org/remote-monitoring).
+It is recommended, but not necessary that the "data interval" of your Gateway is set via Sun Synk Connect to 60s for best latency.
+If you do not have that setting available you can request it set via Sun Synk, your installer or you can ask for an [User Level Access Change Request](https://www.sunsynk.org/remote-monitoring)
+
+This binding uses your Sun Synk Connect credentials to access Sun Synk's web services via an openHAB Bridge (SunSynk Account).
+The bridge manages the account authentication and the discovery of SunSynk Inverter Things. Only the Inverter Thing is currently supported.
+
+Acknowledgements:
+
+- [Power Forum](https://powerforum.co.za/topic/12604-sunsynk-wifi-dongle-hacking/page/3/)
+- [AsTheSeaRises](https://github.com/AsTheSeaRises/SunSynk_API)
+- [jamesridgway](https://github.com/jamesridgway/sunsynk-api-client/tree/main)
+- [kellerza](https://github.com/kellerza/sunsynk)
+
+## Supported Things
+
+|Name            | Thing type    |  Thing Id  |
+|----------------|---------------|------------|
+|SunSynk Account | Bridge Thing  | account    |
+|SunSynk Inverter| Thing         | inverter   |
+
+## Discovery
+
+The binding supports discovery via configuring your login and password in an openHAB bridge.
+1. Add the sunsynk binding.
+1. Add a new thing of type SunSynk Account via the SunSynk Binding and configure with username and password.
+1. Go to Inbox press \[+\] and via the SunSynk Account start discovery \[Scan\] of inverters. 
+1. Inverters should appear in your inbox!
+
+The SunSynk Account bridge thing will discover connected inverters through the UI Scan service.
+When using the UI Scan service all the parameters for an Inverter Thing are discovered.
+
+- Inverter Serial maps to the Sun Synk Connect inverter serial number
+- Inverter Name maps to the Sun Synk Connect inverter alias
+- Refresh time (advanced) default 60s; determines the interval between polls of Sun Synk Connect. A value above 60 is enforced. When setting this remember your inverter values are only published by Sun Synk Connect at the rate set by "data interval".
+
+The refresh rate is limited to once every 60s to prevent too many requests from the Sun Synk Connect API, although there is no rate limit, the Sun Synk data is fully refreshed at the "data interval" set in Sun Synk Connect, at best that is every 60s. This can mean the data in openHAB is more than 1 minute delayed from real-time. Commands sent (from openHAB) to Sun Synk are buffered up until the next refresh interval and as they take a while to propagate through to your inverter, some channels are not refreshed (read back) from Sun Synk Connect until the next minute. 
```

Comment:
```suggestion
The refresh rate is limited to once every 60s to prevent too many requests from the Sun Synk Connect API, although there is no rate limit, the Sun Synk data is fully refreshed at the "data interval" set in Sun Synk Connect, at best that is every 60s.
This can mean the data in openHAB is more than 1 minute delayed from real-time. 
Commands sent (from openHAB) to Sun Synk are buffered up until the next refresh interval and as they take a while to propagate through to your inverter, some channels are not refreshed (read back) from Sun Synk Connect until the next minute.
```

Reply:
added newlines